code/game/machinery/clonepod.dm

```diff
@@ -623,14 +629,36 @@
 
 /obj/machinery/clonepod/emag_act(user)
 	. = ..()
-	eject_clone(TRUE)
+	malfunction()
 	return TRUE
 
 /obj/machinery/clonepod/emp_act(severity)
-	if(prob(50))
-		eject_clone(TRUE)
+	if(prob(100 / (severity * emp_resistance)))
+		malfunction()
 	return ..()
 
+/obj/machinery/clonepod/proc/malfunction()
+	if(clone)
+		var/datum/mind/patient_mind = locateUID(patient_data.mindUID)
+		patient_mind.transfer_to(clone)
+		clone.grab_ghost()
+		var/message
+		message += "<b>Agony blazes across your consciousness as your body is torn apart.</b><br>"
+		message += "<i>Is this what dying is like? Yes it is.</i>"
+		to_chat(clone, "<span class='warning'>[message]</span>")
```

Comment:
If you're gonna split this up like this, might as well list it and .Join("<br>") it, otherwise you can probably just put it into one line

Reply:
moved into one line